### DISCUSSION TITLE
Windows version of spectracle

### DISCUSSION THREAD

**Discussion by Astroghost-Francois** [grillbaer/spectracle#16] [General]
Hi ,
is it possible to have access to the msi installation file of spectracle please ?

Thanks ,

François

**Comment by grillbaer** [comment]
Hi François,
the latest build artifacts timed out, so I triggered a fresh build.
You can find the installer under [Actions](https://github.com/grillbaer/spectracle/actions/workflows/maven.yml) within the latest build or directly here:

https://github.com/grillbaer/spectracle/actions/runs/12014832292/artifacts/2234243498

Best regards,
Holger

**Comment by Astroghost-Francois** [comment]
Hi Holger ,

thanks for your fast answer.
All work fine.

Best regards,

François

----- Mail original -----
De: "Holger Fleischmann" ***@***.***>
À: "grillbaer/spectracle" ***@***.***>
Cc: "BERNIER Francois" ***@***.***>, "Author" ***@***.***>
Envoyé: Lundi 25 Novembre 2024 18:06:16
Objet: Re: [grillbaer/spectracle] Windows version of spectracle (Discussion #16)





Hi François, 
the latest build artifacts timed out, so I triggered a fresh build. 
You can find the installer under Actions within the latest build or directly here: 

https://github.com/grillbaer/spectracle/actions/runs/12014832292/artifacts/2234243498 

Best regards, 
Holger
